service/controller/app/app.go

```diff
@@ -17,14 +17,16 @@ import (
 	"github.com/giantswarm/app-operator/v4/pkg/label"
 	"github.com/giantswarm/app-operator/v4/pkg/project"
 	"github.com/giantswarm/app-operator/v4/service/controller/app/controllercontext"
+	cachedk8sclient "github.com/giantswarm/app-operator/v4/service/internal/k8sclient"
```

Comment:
```suggestion
	"github.com/giantswarm/app-operator/v4/service/internal/k8sclientcache"
```

I found the naming a bit hard to follow here. What about this?

Reply:
Much better!